app/src/main/java/org/simple/clinic/instantsearch/InstantSearchScreen.kt

```diff
@@ -322,3 +339,6 @@ class InstantSearchScreen :
     fun inject(target: InstantSearchScreen)
   }
 }
+
+@Parcelize
+object BlankBpPassport : Parcelable
```

Comment:
We can make this an inner type in `InstantSearchScreen` instead of a global one.